esmvaltool/recipes/examples/recipe_check_obs.yml

```diff
@@ -548,16 +548,16 @@ diagnostics:
       thetao:
         mip: Omon
       no3:
-        mip: Oyr
+        mip: Omon
       o2:
-        mip: Oyr
+        mip: Omon
       po4:
-        mip: Oyr
+        mip: Omon
       si:
-        mip: Oyr
+        mip: Omon
     additional_datasets:
```

Comment:
Shall the recipe include all suported variable types (Omon, Oyr, and Omon surface)? This would help to test that the cmorization worked fine for everything.

Reply:
It basically need only `Omon`, as the other two set of data (Oyr and Omon ocean surface) are derived from this one, but we may think to test the whole set of data ...

Reply:
I forgot that the test checks also the total number of files produced, which accounts for the Omon 3D + Omon os & Oyr 3D. @remi-kazeroni  could  we go along with this?

Reply:
@tomaslovato the test already checks for all these 18 files [here](https://github.com/ESMValGroup/ESMValTool/blob/fbefb8e8329de13a95817a413bc50d8e22b76235/tests/integration/test_cmorize_obs.py#L117). I checked that the tests pass on my machine so this should be fine.